plugins/destination/plugin_testing.go

```diff
@@ -155,28 +158,32 @@ func (s *TestSuite) destinationPluginTestWriteAppend(ctx context.Context, p *Plu
 		return fmt.Errorf("failed to migrate tables: %w", err)
 	}
 
-	sourceName := uuid.NewString()
+	sourceName := "testAppendSource" + uuid.NewString()
 	resource := schema.DestinationResource{
 		TableName: table.Name,
-		Data:      testdata.TestData(),
+		Data:      testdata.GenTestData(table),
 	}
+	_ = resource.Data[0].Set(sourceName)
+	_ = resource.Data[1].Set(syncTime)
 
 	if err := p.writeOne(ctx, tables, sourceName, syncTime, resource); err != nil {
 		return fmt.Errorf("failed to write one second time: %w", err)
 	}
 
-	resource = schema.DestinationResource{
+	resource2 := schema.DestinationResource{
 		TableName: table.Name,
-		Data:      testdata.TestData(),
+		Data:      testdata.GenTestData(table),
 	}
 
 	if !s.tests.SkipSecondAppend {
 		// we dont use time.now because looks like there is some strange
 		// issue on windows machine on github actions where it returns the same thing
 		// for all calls.
-		secondSyncTime := syncTime.Add(time.Second).UTC()
+		secondSyncTime := syncTime.Add(10 * time.Second).UTC()
```

Comment:
Just curious why this had to be increased to 10 seconds?

Reply:
it can be also one second. prob just was debugging something and it was changed. will change it back.